src/database/migrations/20221101124519_create_prices_table.ts

```diff
@@ -4,7 +4,7 @@ import { Knex } from 'knex';
 */
 export async function up(knex: Knex): Promise<void> {
   await knex.schema.createTable('prices', (table) => {
```

Comment:
Voitais tallentaa hinta myös historiaan. Eli tähän tauluun vielä `id` ja jokin parametri, joka kertoo onko hinta juuri nyt aktiivinen. Aktiivisia hintoja voi kaasulle olla vain yksi kerrallaan, joten joku archived käy myös.

Miksi? Että saadaan hintahistoriaa ja täyttötapahtumien maksun muodostus voidaan jäljittää tähän tauluun asti.

Reply:
Ehkä ne vois olla `activeSince` ja `activeEnded` tjsp (@ilesoft  on hyvä nimeämään näitä). Että sinne tulis semmonen aikalinja, milloin just tää hinta on ollut aktiivinen

Reply:
Tuossa taulussa on timestampit päällä, jotenka jokaiseen riviin generoidaan created_at -arvo, jota voidaan käyttää historiatarkasteluun